spec/features/visitors/home_page_spec.rb

```diff
@@ -10,7 +10,7 @@
   #   Then I see "Welcome"
```

Comment:
You've done a great job of writing easily understood, runnable code in the spec. This is a good thing to aim for. I'm assuming you'll keep this up so consider removing the comments from this spec (and other specs) as they are an extra overhead for maintenance that doesn't add anything to what's already there.

(The comments are now stale on line 10 for example).